src/main/resources/rp-start

```diff
@@ -6,18 +6,6 @@
 
 set -e
 
-# Lots of people seem to want to use these images outside of intended
-# platforms, so warn if that's the case
-
-if [ "$RP_PLATFORM" = "" ]; then
-  echo "********************************************************************************"
-  echo "* Unable to detect active platform; this Docker image is intended to be run in *"
-  echo "* a supported orchestration environment such as Kubernetes, not stand-alone.   *"
-  echo "********************************************************************************"
-
-  exit 1
-fi
-
```

Comment:
I removed the default entrypoint, so this isn't needed anymore. I think it's more natural this way -- thus e.g. `docker run <image> /bin/bash` works as expected.